src/interfaces/userInterface.ts

```diff
@@ -0,0 +1,33 @@
+export interface IUser {
+    customer_key: string,
+    phone: string,
+    gender: string,
+    orientation: string,
+    location: {
+        type: string,
+        coordinates: [number]
+    },
+    is_active: boolean,
+    user_name: string,
+    blocked_list: [number],
+    profile_info: {
+        about: string,
+        job_title: string,
+        company: string,
+        university: string,
+        live_in: string,
+        profile_pic: [
+            {
+                url: string,
+                is_default: boolean
+            }
+        ]
+    },
+    email: string,
+    password: string,
```

Comment:
Not planning to keep password authentication. Only hotp or totp based login

Reply:
Okay. Will remove that.

Reply:
I think it would be awesome to have email so that we can send some emails regarding related to match or advertisment.

Reply:
Password can also would be good in case we want to use the facebook, instagram, google login and that is pretty much more probably would be required.

Reply:
Password isn't required for social authentication as well